data/noop/noop.go

```diff
@@ -34,29 +32,43 @@ func (n *noop) PaymentCreate(ctx context.Context, args dpp.PaymentCreateArgs, re
 func (n noop) PaymentRequest(ctx context.Context, args dpp.PaymentRequestArgs) (*dpp.PaymentRequest, error) {
```

Comment:
shouldn't this be called `PaymentTerms` actually?

Reply:
Yes. You are right. I will change it